samples/CallingStatefulBreakoutRooms/package.json

```diff
@@ -0,0 +1,110 @@
+{
+  "name": "@internal/callingstatefulbreakoutrooms",
```

Comment:
Do we need this in our repo? Just wanted to make sure we're OK with extra overhead maintaining this. (Other options would be quickstarts repo, or a github show&tell or Gist)
If we are we should have regression tests somewhere within this directory

Reply:
I see. Good point. If I add it to the quickstarts repo then it would be bound to a specific version of communication react and would only need to be tested when we upgrade it. I will probably move it quickstart repo. Thanks.

Reply:
Another option would be to add it to the stateful app I made for inbound calling. would just need more options on the home screen on what kind of call its joining. (or a new breakoutrooms page)

Reply:
I don't think I should add it to your inbound outbound app because it's supposed to be a simple calling app where I add the bare minimum to get breakout rooms working. I reference the commits in the storybook page to walkthrough the steps.